lib/xml2json.js

```diff
@@ -1477,7 +1477,7 @@ module.exports.xml2json = function (
    */
   function normalizeTarget(target) {
     const open = target.indexOf("(");
-    const close = target.indexOf(")");
+    const close = target.lastIndexOf(")");

```

Comment:
Why is that better? We don't have nested parentheses in annotation targets yet, and if we get them, this code will have to be reworked anyway.

If you found a situation where the old code breaks and the new code works better, please add a test case.